### ISSUE TITLE
Semantic info not always spoken with the default voice

### ISSUE THREAD

**Issue by bart-simons** [FreedomScientific/VFO-standards-support#9]
### Summary

My OS and JAWS are Dutch. Automatic language detection is switched on. Synthesizer is vocalizer expressive.
When reading a page with lang="en", its contents is correctly spoken with the English voice. JAWS announces the role of elements such as link, image, heading level. This is nicely done in Dutch and with the default Dutch voice. This is true as long as this announcement preceeds the actual text. But when it is behind, then it is spoken by the English voice that is used to speak the contents.

### Example

1. Go to https://www.gov.uk/
2. arrow down until you reach the first heading level 1
3. JAWS reads with the Dutch voice "kopniveau 1" (which is the correct Dutch translation of heading level 1), followed by the actual text of the heading "welcome to gov.uk. The latter is spoken with the English voice. So this is how it should be.
4. Go back to top with ctrl+home and press h to reach the same heading.
5. Now you hear "welcome to gov.uk, kopniveau 1". The problem is that both parts are spoken with the English voice. It should switch to the Dutch voice to speak "kopniveau 1".

This seems consistent for semantic info that is spoken after the actual text. For some reason JAWS announces the word link before reading the link text but it reads the value of a button and after that it says that it is a button. Since the link announcement is before, the voice is switched corrrectly. Since the button announcement is at the end, there is no voice swapping and the word knop (translation of button) is spoken with the English voice.

### Expected result

JAWS to use the correct voice to speak its own information (such as button, link, heading level, tab selected, checkbox not checked etc). It should not matter whether this info is at the beginning of the string or at the end.

### Actual result

Only correct when the JAWS info is read first. When the info follows other text from the web page, JAWS keeps talking with the voice that was used to speak the contents.

#### JAWS version and build number

JAWS 18.0.4104 (but the issue exists since a long time. It gets more and more annoying with all the new aria announcements)

#### Operating System and version

Windows 7 64-bit

#### Browser and version:

IE11 and also in Firefox 56 (but older versions as well)

**Comment by stevefaulkner** [comment]
thanks @bart-simons will look into both of these related issues.

**Comment by hanshillen** [comment]
Confirmed,

I have created a test case and have filed this issue (combined with #8) internally as bug 97133: "Dutch JAWS does uses incorrect voice when announcing English content".

**Comment by dfarough** [comment]
Probably related:
I saw this on Webaim list today.
A link coded with language  of French. the link text is spoken with the French voice as expected, but the title text is spoken using the English voice. 
Example code was:
<a lang="fr-fr" title="Aller à la page 2" href="http://google.ca/">2</a>

**Comment by bart-simons** [comment]
@dfarough yes this is a combination of both issues. It reads 2 with the French voice, then "bezocht link" (Dutch translation of visited link) with the French voice (instead of Dutch), and then the title text with the Dutch voice (instead of French).

**Comment by stevefaulkner** [comment]
@hanshillen can you check with latest JAWS to see if this is still an issue?

**Comment by hanshillen** [comment]
I did some testing with JAWS 2023.2307.37. The issue is still occurring: in the scenario described above, JAWS announces "kopniveau X" with the Dutch voice when it's prefixed to the heading text (e.g. when using arrow keys navigation), and with the English voice when it comes after the heading text (e.g. when using heading navigation).

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 40481

**Comment by JeremiahRogersVispero** [comment]
@bart-simons, I recognize this is a very old issue, but if you could be so kind as to confirm if it is still happening with JAWS 2026.2603.86, I would appreciate it. I tried, today, and I could not reproduce. Thanks in advance.

**Comment by bart-simons** [comment]
Hello,

Thanks for coming back to this and allowing me to update.
It is solved in most cases.

Steps to reproduce:
I use the Dutch JAWS version.
I go to a page like gov.uk which has the language set to English. JAWS speech switches to English to read the content from the web page.
Press h to navigate by heading
The first heading is “navigation menu” and this is followed by the semantics announcement kop niveau 2 and it is spoken with the Dutch voice as it should
Next is the heading level 1 which is also spoken correctly: english text of heading with the English voice and semantic info rrom JAWS in Dutch spoken with the dutch voice
Next heading is “popular on gov.uk: here the reported problem remains: the text is spoken with the English voice. In this case kop  niveau 2 is spoken with the English voice instead of the Dutch voice.
Same problem for the next one services and information
also for all headings level 3 that follow. For all of them kop niveau 3 is spoken with the English voice and it should be the Dutch voice.
If you keep pressing the letter h for heading navigation, we land on a heading called “featured” and here is no problem: kop niveau 2 is spoken with the Dutch voice as it should.

Unfurtunately I see no pattern why the problem persist for some headings and not for others.

I also tried French by going to paris.fr
Here is the same problem when navigating by heading
Ville de paris kop niveau 1 (correctly spoken with first the French and than switched to dutch voice)
Accès rapides: not correct: kop niveau 2 is spoken with French voice. It should switch to Dutch for the semantic announcement
Actualité: same problem
Next headings are ok again

I hope you discover a pattern and find a way to fix the announcements consistently

Thanks a lot

I am using the most recent JAWS build that you referenced, Windows 11 and Chrome

Bart

Van: Jeremiah Rogers ***@***.***>
Verzonden: maandag 20 april 2026 23:44
Aan: FreedomScientific/standards-support ***@***.***>
CC: Bart Simons ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [FreedomScientific/standards-support] Semantic info not always spoken with the default voice (#9)

[Afbeelding verwijderd door afzender.]JeremiahRogersVispero left a comment (FreedomScientific/standards-support#9)<https://github.com/FreedomScientific/standards-support/issues/9#issuecomment-4284479672>

@bart-simons<https://github.com/bart-simons>, I recognize this is a very old issue, but if you could be so kind as to confirm if it is still happening with JAWS 2026.2603.86, I would appreciate it. I tried, today, and I could not reproduce. Thanks in advance.
